content/en/docs/tutorials/acme/ingress.md

```diff
@@ -26,7 +26,7 @@ $ brew install kubernetes-helm
 
 ## Step 1 - Install Tiller
 
-> *Skip this section if you have Tiller set-up.*
+> *Skip this section if you have Tiller set-up. Only for helm v2*
```

Comment:
Can we make it `Helm` with a capital H?

Reply:
sure. Sure I change L317 also ?
https://github.com/cert-manager/website/blame/7416ac6d9c8e6f0ffe771d6bc0b7566b9034d015/content/en/docs/tutorials/acme/ingress.md#L317

Reply:
Always accepting a PR for that change!